gcloud/datastore/batch.py

```diff
@@ -279,3 +271,35 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 self.rollback()
         finally:
             _BATCHES.pop()
+
+
+def _assign_entity_to_mutation(mutation_pb, entity):
+    """Helper method for ``Batch.put``."""
+    auto_id = entity.key.is_partial
+
+    key_pb = entity.key.to_protobuf()
+    key_pb = helpers._prepare_key_for_request(key_pb)
+
+    if auto_id:
+        insert = mutation_pb.insert_auto_id.add()
+    else:
+        insert = mutation_pb.upsert.add()
```

Comment:
Can you add a comment above here explaining that we don't use `insert` or `update` but are aware of their existence and may add in future?